models/exchange/binance/api.py

```diff
@@ -65,7 +67,17 @@ def __init__(self, api_key: str='', api_secret: str='', api_url: str='https://ap
         self.api_url = api_url
         self.api_key = api_key
         self.api_secret = api_secret
-        self.client = Client(self.api_key, self.api_secret, { 'verify': False, 'timeout': 20 })
```

Comment:
Also wondering if there are latency issues with Binance to maybe increase the "timeout" to maybe 60. I mean 20 seconds is long! But if the exchange is struggling maybe giving it a bit more time is better.